### PR TITLE
Community page update

### PR DESCRIPTION
**Pull request by lsqproduction**
Add link to Github discussion.
Remove Gitter and Spectrum.
There are still some old blogs and guides with spectrum and gitter, do we want to replace those as well?

### CODE REVIEW

**Review by kevinbluer — CHANGES_REQUESTED** [review]
Positioning of the "Connect on Github Discussions" button looks a little off to me

Screen Shot 2022-04-29 at 2 38 44 PM

File: custom_theme/community.html [hunk 1]
```diff
@@ -57,30 +57,19 @@ <h1><img class="support-heading" src="../img/support-header.svg" alt="Life Prese
           <div class="col">
             <div class="card">
               <div class="card-body card-zendesk">
-                <h2>Open a Support Ticket! <span class="badge rounded-pill bg-success">NEW</span></h2>
-                <p>The easiest way to track your conversation and contribute to our knowledge base! Open a support ticket to begin talking to our support staff. Common questions will be added to our knowledge base to assist future Trufflers!</p>
-                <a target="_blank" rel="noopener noreferrer" href="https://trufflesuite.zendesk.com/hc/en-us/requests/new" class="btn btn-sm btn-truffle text-uppercase">Open a Support Ticket</a>
+                <h2>Join The Conversation</h2>
```
Comment (kevinbluer) [inline custom_theme/community.html:60]: Thoughts on changing this to "Join the Discussion" to be more inline with the fact it's Github Discussions?

**Comment by kevinbluer** [comment]
I feel like the spacing before the following headers feels a bit excessive. This might be more of a site-wide styling rule although should be easy enough to override at the page level if you agree.

Screen Shot 2022-04-29 at 2 52 17 PM

File: custom_theme/community.html (hunk 1 above)
Comment (lsqproduction) [reply custom_theme/community.html]: Agree. After syncing with @emilyJLin95, I did a revamp of the page hoping for a more simplified and organized look.

**Review by emilyJLin95 — APPROVED** [review]
LGTM
I do wonder if we should consider renaming Community to Support? I'll leave it to others to decide

**Comment by realcliffzhang** [comment]
@lsqproduction I am still able to find some anchors to Gitter and Spectrum on some blogs and guides, we may want to replace them with GitHub Discussions just in case someone gets redirected there. Otherwise lgtm!

**Comment by leeftk** [comment]
LGTM

**Review by leeftk — APPROVED** [review]

**Pull request merged**